pkg/scheduler/metrics/metrics.go

```diff
@@ -377,6 +383,15 @@ func InitMetrics() {
 		},
 		[]string{"call_type"})
 
+	ResourceClaimCreatesTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "resourceclaim_creates_total",
+			Help:           "Number of ResourceClaims creation requests within scheduler for extended resource",
```

Comment:
Maybe don't suggest the specific use case, so we could extend it in the future, if needed
```suggestion
			Help:           "Number of ResourceClaims creation requests within scheduler",
```